src/parser.js

```diff
@@ -124,12 +153,14 @@ export default class Parser {
                     break;
                 }
                 if (lastAdded) {
-                    let spaceProp = `spaces.${lastAdded}.after`;
-                    dotProp.set(node, spaceProp, dotProp.get(node, spaceProp, '') + content);
-                    let commentProp = `raws.spaces.${lastAdded}.after`;
-                    let existingComment = dotProp.get(node, commentProp);
+                    ensureObject(node, 'spaces', lastAdded);
```

Comment:
I'm curious why you decided that `ensureObject` shouldn't return the last object.